src/datasets/other/newtab/reference.md

```diff
@@ -0,0 +1,83 @@
+# Newtab Interactions
+
+
+## Introduction
+The `telemetry.newtab_interactions` dataset is useful for analyzing data on the New Tab on Firefox Desktop.  It contains a single source for user interactions with the New Tab and allows analysis to be done at either a client or visit level.  This table contains data from the Glean ["newtab" ping](https://dictionary.telemetry.mozilla.org/apps/firefox_desktop/pings/newtab). 
+
+## Content 
+This dataset can contain more than one row for each  `submission_date`, `client_id` and `visit_id`.  Each unique tab opened is defined as a unique New Tab `visit_id`.  We recommend aggregating by `client_id`, `submission_date` and `visit_id`. 
+
+The `client_id` is the Glean client identifier, there is also the `legacy_telemetry_client_id` which can be used to join this data to non-Glean data tables. 
+
+## Background and Caveats
+### Scheduling:  
+"newtab" pings can be sent for one of two reasons. 
+component_init: The newtab component init'd, and the newtab and homepage settings have been categorized. This is mostly to ensure we hear at least once from clients configured to not show a newtab UI.
+newtab_session_end: The newtab visit ended. Could be by navigation, being closed, etc. 
+
+This dataset does not have a long history. The newtab ping went live in Fall 2022 and therefore we do not have data prior to that launch.  Any historical analysis will not be possible until we collect more data. 
+
+Currently, telemetry is not yet instrumented for organic topsites tiles. This work is in progress and will be added to the dataset soon.  For now, all total topsites metrics (impression, clicks) only include sponsored topsites data. 
+
+Some of the preference settings enabling features like Pocket and Topsites can be set to `enabled` in regions where these features are not available. In these cases, it is best to filter by country to only include regions where these features are available. In Looker, there are pre-set country groups for Topsties Available and Pocket Available to make this easier. 
+
+## Data Reference 
+
+### Field Descriptions 
```

Comment:
We should add these directly to the schema as well. Pages like this tend to become outdated somewhat quickly